### PR TITLE
CHORES: Add procedures to configure WSL 2

### LINKED ISSUE #4: Add procedures to configure WSL 2
Currently no procedures to explain how to use WSL2 in Windows environment.

1. Instruction to prepare [Windows 10](https://docs.microsoft.com/en-us/windows/wsl/install-win10) to run WSL 2
1. [A Linux Dev Environment on Windows with WSL 2, Docker Desktop and More](https://nickjanetakis.com/blog/a-linux-dev-environment-on-windows-with-wsl-2-docker-desktop-and-more)  
1. [Using Docker in Windows for WSL2](https://code.visualstudio.com/blogs/2020/03/02/docker-in-wsl2)
1. Add [URL](https://code.visualstudio.com/blogs/2020/03/02/docker-in-wsl2) that refers to how to configure WSL 2

### PR DESCRIPTION
**Pull request by dixonsiu**
Closes #4.

### CODE REVIEW

**Comment by j1n6** [comment]
When I looked at the docker repo last week, initially I thought it contains all the personium services.
Would it be good to clarify what services  (e.g. Unit/Cell/UnitManager) are included? Especially the repo name suggest it includes all?

**Review by j1n6 — APPROVED** [review]

**Comment by dixonsiu** [comment]
> When I looked at the docker repo last week, initially I thought it contains all the personium services.
> Would it be good to clarify what services (e.g. Unit/Cell/UnitManager) are included? Especially the repo name suggest it includes all?

Good idea. I will respond to your suggestion in Issue #3 .

File: README.md [hunk 1]
```diff
@@ -2,10 +2,16 @@
...
+The followings must be installed and configured in advance to build and run Personium Unit offered from this repository.
 
-* docker
-* docker-compose
+* [Docker](https://docs.docker.com/get-docker/)
+* [Docker Compose](https://docs.docker.com/compose/install/)
+
+### Windows 10 User  
+Special requirements if you want to run Docker in Windows 10. Follow the [official instructions from Microsoft](https://docs.microsoft.com/en-us/windows/wsl/install-win10) to install WSL 2.  
+> I highly recommend to check out [A Linux Dev Environment on Windows with WSL 2, Docker Desktop and More](https://nickjanetakis.com/blog/a-linux-dev-environment-on-windows-with-wsl-2-docker-desktop-and-more) before you start installing anything.
```
Comment (yoh1496) [inline README.md:12]: [A Linux Dev Environment on Windows with WSL 2, Docker Desktop and More](https://docs.microsoft.com/en-us/windows/wsl/install-win10) looks nice for person who want to know what WSL2 makes available to use.
But it seems to contains stuffs not related to developing personium, so I don't think it is `highly recommended` for personium beginners 🤔

**Review by yoh1496 — APPROVED** [review]
Thanks dixon for writing starting with Docker in WSL2. I think it is ok.

**Pull request merged**